aws-privatelink/dns-endpoints.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+#
+# dns-endpoints.sh
+#
+# Output zone records to correctly map to zonal endpoints for Confluent Cloud.
+#
+# Example:
+#
+#   % ./dns-endpoints.sh vpce-0123456789abcdef0
+#     *                         CNAME vpce-0123456789abcdef0-01234567.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
+#     *.usw2-az2                CNAME vpce-0123456789abcdef0-01234567-us-west-2b.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
+#     *.usw2-az1                CNAME vpce-0123456789abcdef0-01234567-us-west-2a.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
+#     *.usw2-az3                CNAME vpce-0123456789abcdef0-01234567-us-west-2c.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
+#
+
+set -eu -o pipefail
+
+if [[ $# != 1 ]]; then
+    echo "usage: $0 <vpce-0123456789abcdef0>" 1>&2
```

Comment:
that doesnt explain what the arg is supposed to represent

Reply:
```
+++ aws-privatelink/dns-endpoints.sh
@@ -17,7 +17,9 @@
 set -eu -o pipefail

 if [[ $# != 1 ]]; then
-    echo "usage: $0 <vpce-0123456789abcdef0>" 1>&2
+    echo "usage: $0 <VPC Endpoint>" 1>&2
+    echo ""
+    echo "example: $0 vpce-0123456789abcdef0"
     exit 1
 fi

```

Fixed.